src/components/Quotes.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+
+function Quotes() {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    const send = () => {
+      setLoading(true);
+      try {
+        fetch('https://api.api-ninjas.com/v1/quotes?category=money', {
+          method: 'GET',
+          headers: { 'X-Api-Key': 'tTySwiau1PsXJs5wvhf1dw==NK8bNzv70yBdLNYz' },
+          contentType: 'application/json',
+        })
+          .then((response) => response.json())
+          .then((json) => setData(json));
+      } catch {
+        setHasError(true);
+      }
+      setLoading(false);
+    };
+    send();
+  }, [setData, setLoading]);
```

Comment:
- Optional
Including `setData` and `setLoading` in the dependencies array is unnecessary. The reason is that these functions are predefined in the component's scope and will never change during the component's lifetime. Consider removing them.
- Also, you don't need to have a separate function `send` and then call that at the end. When ever, `useEffect` runs, it will run what ever code you put inside it. In this case you can consider removing the send function and putting all the code inside send function directly inside the `useEffect`.